data-publisher/src/main/kotlin/no/nav/amt/tiltak/data_publisher/model/DeltakerlistePublishDto.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.amt.tiltak.data_publisher.model
+
+import no.nav.common.json.JsonUtils
+import org.springframework.util.DigestUtils
+import java.time.LocalDate
+import java.util.*
+
+data class DeltakerlistePublishDto(
+	val id: UUID,
+	val arrangorId: UUID,
+	val type: String,
+	val navn: String,
+	val status: String,
+	val tiltakNavn: String,
```

Comment:
Når vi har med tiltaknavn her bør vi kanskje ha med arrangørnavn også? Jeg tror ikke det er noen steder der vi ikke har behov for arrangørnavn når vi skal vise en deltakerliste :)